script.js

```diff
@@ -0,0 +1,18 @@
+const menuBtn = document.querySelector('.hamburger');
+const menu = document.getElementById('theMenu');
+const xBtn = document.getElementById('xBtn');
+const navLinks = document.querySelectorAll('.navs');
+
+function openMenu() {
+  menu.style.opacity = '90%';
+}
+
+function closeMenu() {
+  menu.style.opacity = 0;
+}
+
+menuBtn.addEventListener('click', openMenu);
+navLinks[0].addEventListener('click', closeMenu);
+navLinks[1].addEventListener('click', closeMenu);
+navLinks[2].addEventListener('click', closeMenu);
```

Comment:
- [ ] This also looks kind of repetitive! 🤔 You're implementing the exact same operation but just changing the index of the elements inside an array. This is the perfect use case for a loop!

```suggestion
navLinks.forEach((navLink) => {
  navLink.addEventListener('click', closeMenu)
})
```

This way, you're programmatically adding event listeners to each element inside the `navLinks` array, so in the future, if you want to add more links, you don't have to add a new line adding a listener to the `navLinks[3]` element, this loop will take care of that! 🤓 🔧